kong/db/dao/init.lua

```diff
@@ -482,15 +482,14 @@ end
 
 
 local function check_update(self, key, entity, options, name)
-  local transform
-  if options ~= nil then
-    local ok, errors = validate_options_value(self, options)
-    if not ok then
-      local err_t = self.errors:invalid_options(errors)
-      return nil, nil, tostring(err_t), err_t
-    end
-    transform = options.transform
+  options = options or {}
```

Comment:
I've just moved `options = options or {}` higher in the code and that allowed me to unnest this block & if. Turning on `hide_whitespaces` in GH makes it easier to review